waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -562,13 +555,28 @@ when defined(rlnzerokit):
 
     return ok(true)
 
-  proc insertMember*(rlnInstance: ptr RLN, idComm: IDCommitment): bool =
```

Comment:
Why should we replace this one entirely? it is a useful utility function and can coexist with `insertMemebers`.

Reply:
ditto

Reply:
Addressed in 37bd29fb